raiden/tasks.py

```diff
@@ -139,6 +139,10 @@ def _run(self):  # pylint: disable=method-hidden
         finally:
             self.callbacks = list()
 
+    def is_primed(self):
```

Comment:
Nitpick: I would use a simpler term understandable by developers... i was confused there for a minute as to what `primed` meant... can i suggest to change this to `is_initialized` or `has_started`?